src/ios/SupportKitCordova.m

```diff
@@ -12,13 +12,17 @@
 @implementation SupportKitCordova
 
 - (void)init:(CDVInvokedUrlCommand *)command {
-    NSDictionary *settings = [command argumentAtIndex:0];
+    NSMutableDictionary *settings = [command argumentAtIndex:0];
```

Comment:
Is the return value of `argumentAtIndex` guaranteed to be mutable? Or should you create a mutable copy first?